src/server/pachyderm_test.go

```diff
@@ -10449,8 +10447,7 @@ func TestInterruptedUpdatePipelineInTransaction(t *testing.T) {
 	require.NoError(t, createPipeline(c, inputC, true))
 
 	_, err = c.FinishTransaction(txn)
-	require.YesError(t, err)
-	require.Matches(t, "outside of transaction", err.Error())
+	require.NoError(t, err)
```

Comment:
Since we're expecting this to succeed now, we should probably check that the latest pipeline info is the right version and has the right input.